carbon_bomb.py

```diff
@@ -1353,12 +1353,7 @@ def create_carbon_bombs_table():
 
     df_carbon_bombs = get_information_from_GEM(df_carbon_bombs)
     df_carbon_bombs = complete_GEM_with_ChatGPT(df_carbon_bombs)
-
-    # Add manualy EACOP carbon bomb
     df_carbon_bombs["Bomb_type"] = "Extraction"
```

Comment:
Do we want to keep Bomb_type in the table ? I think it is useless now and would bring more confusion on our data